### ISSUE TITLE
Maybe re-add the default aria-current for active routes

### ISSUE THREAD

**Issue by webJose** [WJSoftware/svelte-router-core#107]
It is kind of a pain to keep adding `aria-current` explicitly after losing this default behavior in **v0.11.0**.  This change was motivated by the fact that sometimes we should be setting `aria-selected` and not `aria-current`.

To revive this default, make sure it deactivates itself if any aria property is explicitly specified.
